data/remnant/remnant missions.txt

```diff
@@ -1060,6 +1060,326 @@ mission "Remnant: Learn Sign Follow Up"
+		fleet
+			names "remnant"
+			variant
+				"Rescue Dummy"
+		dialog
+			`As you enter the system, your sensors pick up a lot of debris. Further analysis confirms your worst fears. The <npc> is destroyed. You frantically spend the next few hours searching for survivors, your hope sinking with every minute that passes.`
+	on complete
+		"rescue missions" ++
+		payment 300000
+		dialog
+			`You land, and with sorrow hanging heavy on your heart, report that the <npc> was lost.`
+	on visit
+		dialog
+			`You have returned, yet the <npc> is still missing.`
+	
+			
+mission "Remnant Rescue 3"
+	job
+	repeat
+	name "Search and Rescue: Supply Ship"
+	description "A ship, the <npc>, has gone missing on a supply run in the Ember Wastes. Find them and help them return to <destination>."
+	source
+		government "Remnant"
+	to offer
+		has "Remnant: Learn Sign Follow Up: offered"
+		random < 30
+	npc accompany save
+		system
+			attributes "ember waste"
+			not attributes "ka'het"
+			not attributes "inaccessible"
+			not attributes "graveyard"
+			not government "Remnant"
+			not system "Perfica"
+			not system "Farinus"
+		personality derelict timid plunders
+		government "Remnant"
+		fleet
+			names "remnant"
+			variant 7
+				Pelican
+			variant 2
+				Gull
+	on complete
+		"rescue missions" ++
+		payment 300000
+		dialog
+			`Your communicator notifies you of a resource allocation of <payment>, and also a thank you from the captain of the <npc>.`
+	on visit
+		dialog
+			`You have returned, yet the <npc> is still missing.`
+	
+	
+mission "Remnant Rescue 3B"
+	job
+	repeat
+	name "Search and Rescue: Supply Ship"
+	description "A ship, the <npc>, has gone missing on a supply run in the Ember Wastes. Find them and help them return to <destination>."
+	source
+		government "Remnant"
+	to offer
+		has "Remnant: Learn Sign Follow Up: offered"
+		random < 5
+	npc kill
+		system
+			attributes "ember waste"
+			not attributes "ka'het"
+			not attributes "inaccessible"
+			not attributes "graveyard"
+			not government "Remnant"
+			not system "Perfica"
+			not system "Farinus"
+		personality staying heroic nemesis unconstrained
+		government "Test Dummy"
+		fleet
+			names "remnant"
+			variant
+				"Rescue Dummy"
+		dialog
+			`As you enter the system, your sensors pick up a lot of debris. Further analysis confirms your worst fears. The <npc> is destroyed. You frantically spend the next few hours searching for survivors, your hope sinking with every minute that passes.`
+	on complete
+		"rescue missions" ++
+		payment 300000
+		dialog
+			`You land, and with sorrow hanging heavy on your heart, report that the <npc> was lost.`
+	on visit
+		dialog
+			`You have returned, yet the <npc> is still missing.`
+
+	
+mission "Remnant Rescue 4"
+	job
+	repeat
+	name "Distress Signal: Graveyard"
+	description "A distress signal has been received from the <npc> in the Graveyard. Hurry there, find them, and help them return safely to <destination>."
```

Comment:
Would "Within the Graveyard" be better...

Reply:
Also, the repetition of "them," but I'm not touching it.

Reply:
I'm honestly not sure on in vs within in this case. As for the "them" repetition, I'll probably have to restructure the sentence, and at the very least, the "them" is actually in the wrong spot. Oops.